glibc_version/src/lib.rs

```diff
@@ -4,7 +4,7 @@
 //!
 //! # Example
 //!
-//! ```
+//! ```no_run
```

Comment:
Can't run this on MacOS, and wasn't sure how to opt out of doc test based on platform.